test/runnable/cdcmp.sh

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 
+export LC_ALL=en_US
```

Comment:
How about using `LC_ALL=C` instead? For example, containers may not have en_US.

Reply:
Yeah but you can suggest it as a change maybe, I have made the PR online. So just after my export sput `export LC_ALL=C`.

Reply:
I can do it online too actually

Reply:
okay done, that works too.